easytab.h

```diff
@@ -834,16 +842,17 @@ EasyTabResult EasyTab_Load_Ex(HWND Window,
 
         if (TrackingMode == EASYTAB_TRACKING_MODE_RELATIVE)
         {
+            EasyTab->IsRelative = 1;
             LogContext.lcPktMode |= PK_X | PK_Y; // TODO: Should this be included in the
                                                  //       PACKETMODE macro define up top?
             LogContext.lcSysMode = 1;
             if (MoveCursor)
             {
-                LogContext.lcSysSensX = LogContext.lcSysSensY = RelativeModeSensitivity;
+                LogContext.lcSysSensX = LogContext.lcSysSensY = (FIX32)(RelativeModeSensitivity*0x10000);
```

Comment:
Could you replace these two FIX32s with an uint32_t instead? I try to avoid using the unnecessary typedefs in the actual implementation of EasyTab, even though all the typedefs are present in the header for prototyping purposes when referring to the docs.